.pre-commit-hooks.yaml

```diff
@@ -18,8 +23,10 @@
   stages: [push]
   pass_filenames: false
```

Comment:
:nail_care:  Nitpicking: Maybe we can put the `pass_filenames` at the end of the defined keys here for consistency? 
(Given the others follow the same order: `id`, `name`,...,etc.).

Reply:
That's the only `pass_filenames` line I didn't touch :)

Anyway, moved it for consistency.